docs/source/technical_tutorials/search/cmr_search_in_r.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>What are the assumptions we are making about our users? Is it a fair assumption everyone understands the concept of a granule? Can we either provide context about them and/or reference out to other content? For example, this <a href="https://docs.maap-project.org/en/latest/technical_tutorials/search/granules.html" target="_blank">notebook</a> has this in the description: <em>Granules are individual files from a sensor where a group of granules make a collection within CMR. The granules are the raw data that will be used for processing.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
+1 we should link out to NASA CMR documentation, Openscapes, and earthaccess
It's not our role to explain CMR in depth, this is just about showing how to do it on MAAP which might differ from other platforms.